include/ChaiVM/interpreter/code-manager.hpp

```diff
@@ -14,20 +14,40 @@ namespace chai::interpreter {
 /**
  * Class to manage bytecode. Similar to ClassLoader.
  */
-class CodeManager {
+class CodeManager final {
 public:
+    static constexpr int8_t CNST_I64 = 'l';
+    static constexpr int8_t CNST_F64 = 'd';
+
+    /**
+     * Loads one instruction.
+     * @param bytecode Raw instruction.
+     */
     void load(bytecode_t bytecode);
 
+    /**
+     * Loads stream of instructions only.
+     * @param istream
+     */
     void load(std::istream &istream);
```

Comment:
@levBagryansky maybe it is better to rename it to `loadCode`? Documentation saves the day, but usage in code looks strange:
```cpp
  loadPool(input_file);
  load(input_file);
  input_file.close();
```
code from `void load(const std::filesystem::path &path);`

Reply:
@c71n93 done